.devcontainer/post-create.fish

```diff
@@ -39,33 +39,35 @@ end
 if test -e package.json
     corepack enable
     corepack prepare pnpm@latest --activate
+    pnpm setup
 
-    # Same thing as running "pnpm setup", but written in fish.
-    # Needed for global CLIs & filesystem file permission issues.
+    # Same thing as running those commands given after
+    # done running `pnpm setup`, but rewritten for fish shell.
+    # Needed for globally-installed CLI apps & assoc. perm issues.
     set -Ux PNPM_HOME $HOME/.local/share/pnpm
     echo 'set -Ux PNPM_HOME "$HOME/.local/share/pnpm"' >> ~/.config/fish/config.fish
```

Comment:
```suggestion
    set -gx PNPM_HOME $HOME/.local/share/pnpm
    echo 'set -gx PNPM_HOME "$HOME/.local/share/pnpm"' >> ~/.config/fish/config.fish
```

Reply:
i am not exactly sure what the difference is; like is you have multiple shells or something?

Reply:
![image](https://github.com/OpenINF/openinf.github.io/assets/17770407/2b8653b0-6a39-4c5c-bb7b-6d4f45dca52f)

Reply:
![image](https://github.com/OpenINF/openinf.github.io/assets/17770407/4f4d9d45-6e2d-4819-a7fc-64a888fb1476)